doc/api/v8.md

```diff
@@ -271,6 +271,89 @@ buffers and external strings.
 }
 ```
 
+## `v8.getCppHeapStatistics(detailLevel)`
+
+Retrieves [CppHeap][] regarding memory consumption and
+utilization statistics using the V8
+[`CollectStatistics()`][] function which may change from one V8 version to the
+next.
+
+* `detailLevel` (`'brief'`|`'detailed`): Specifies the level of detail in the
+  returned statistics.
+  * `brief`:  Brief statistics contain only the top-level allocated and used
+    memory statistics for the entire heap.
+  * `detailed`: Detailed statistics also contain a break down per space and
+    page, as well as freelist statistics and object type histograms.
+
+It returns an object with a structure similar to the
+[`cppgc::HeapStatistics`][] object. You can learn more about the properties of the object in
+the [V8 documentation][`cppgc::HeapStatistics struct`].
```

Comment:
```suggestion
[`cppgc::HeapStatistics`][] object. See the [V8 documentation][`cppgc::HeapStatistics struct`]
for more information about the properties of the object.
```

(I am fairly certain that we still try to avoid "you" as much as possible in the docs..)